superset/views/core.py

```diff
@@ -769,7 +770,8 @@ def save_or_overwrite_slice(
     @api
     @has_access_api
     @expose("/warm_up_cache/", methods=("GET",))
-    def warm_up_cache(  # pylint: disable=too-many-locals,no-self-use
+    @deprecated(new_target="api/v1/chart/warm_up_cache/")
```

Comment:
@michael-s-molina I gather it's now too late to remove this endpoint in 3.0 given there's a new target.

Reply:
unfortunately yes

Reply:
Let's remove it in 4.0 given that the breaking window is closed.